application/controllers/Admin.php

```diff
@@ -0,0 +1,154 @@
+<?php
```

Comment:
```suggestion
<?php
defined('BASEPATH') OR exit('No direct script access allowed');
 /*
 * ADMIN PRIVILEGES : 
 * get
 *  products (all / unit)
 *  customers (all / unit)
 *  orders  (all / unit)
 *  order with its details (orderdetails)  (all / unit)
 *  shoppingcarts (all / unit)
 *  shoppingcart with its details (shoppingcartdetails)  (all / unit)
 *  admin profile(all / unit) ?not sure
 * 
 * insert
 *  products
 * 
 * update
 *  products
 *  orders
 * 
 * */ 
 /* *
 * HOW TO ADD RELATIONAL TABLE (LIKE ORDERS & SHOPPINGCARTS)
 * -> insert data ke table orders | shoppingcarts.
 * -> get id of the latest orders | shoppingcarts update (using $db->insert_id; , src : https://www.w3schools.com/php/php_mysql_insert_lastid.asp)
 * -> insert datadetail + xxxId into xxxdetails.
 * 
 * just my initial idea. Hope it werks!
 * */
class Admin extends CI_Controller {
```

steve nanya dong, ini km memang ganti jd bgini lagi apa gimana yaa?? hehehe, kea smua filenya ditaruh diluar folder admin dan nambahin controllerya di file ini??

Reply:
soalnya pas awal aku pisahin controllernya per page gitu supaya bisa dibedain dalem 1 page itu pake controllernya yg mana aja

Reply:
oh ia aku yg taruh di luar soalnya bingung mo manggil controllernya dari dalam subfolder

Reply:
bisaa kok stevee, jadi pertama" kamu ganti dulu isi ```base_url``` di  ```config/config.php``` nya jadi folder file kamu sbelum folder application 

e.g  :: kalo file pemweb aku, aku taruh di ```../XAMPP/htdocs/Infinistyle/```, which is di dalam folder ```Infinistyle ini nanti baru ada application dan folder" lainnya.  

jadi ```base_url``` nya ini aku ganti jadi ```localhost/Infinistyle``` . 

lalu akses di browsernya, kamu tinggal tulis ```localhost/[nama_folder_controller]/[nama_class_controller]/[nama_method]```

contoh : 
- ```localhost/admin/products/``` <- utk akses halaman productnya admin
- ```localhost/admin/products/add_product/``` <- dipake kalo misalnya admin mau add product